vmdb/db/migrate/20150401090146_add_systemd_and_openstack_related_columns_to_system_services.rb

```diff
@@ -0,0 +1,9 @@
+class AddSystemdAndOpenstackRelatedColumnsToSystemServices < ActiveRecord::Migration
+  def change
+    add_column :system_services, :load, :string
+    add_column :system_services, :active, :string
+    add_column :system_services, :sub, :string
+    add_column :system_services, :openstack_host_service_id, :bigint
+    add_index  :system_services, :openstack_host_service_id
```

Comment:
@Fryguy I actually took the name froms systemctl headers, I guess if it will be this way it could be prefixed so it is not figting ruby. :-)

@roliveri I guess serialized approach should be fine. Although SQL queries will look weird after that, we will need LIKE for every match basically, that also means less effective queries, so I might need roll-ups on the way up. We will query this table a lot for status of OpenStack services. I guess I could encode it to 'running' col, to at least see what is running/failed, even when there are more states.
I added more columns, cause I've seen for windows there are lots of specific columns already .


Btw. any reason why this is not done via STI?